web/docs/advanced/deployment/manually.md

```diff
@@ -265,10 +265,86 @@ netlify deploy --prod
 
 That is it! Your client should be live at `https://<app-name>.netlify.app` ✨
 
+:::caution Redirecting URLs toward `index.html`
+If you follow the instructions above, Netlify will use `netlify.toml` file that Wasp generates by default in `.wasp/build/web-app/`. This will correctly configure Netlify to redirect URLs toward `index.html`, which is important since Wasp is SPA.
+
+If you instead use another method of deployment to Netlify, e.g. do it via CI, make sure that Netlify picks up that `netlify.toml` file, or configure URL redirecting yourself manually on Netlify.
+
+It is recommended to deploy through the Netlify CLI in Github Actions. The first deploy can be through the website or manually to get a `NETLIFY_SITE_ID`, the following deploys can then be automatic.
+:::
+
 :::note
 Make sure you set this URL as the `WASP_WEB_CLIENT_URL` environment variable in your server hosting environment (e.g., Fly.io or Heroku).
 :::
 
+### Deploying through Github Actions
```

Comment:
@infomiho I just merged this PR by a contributor. They made it clear that if deploying on Netlify, we need redirectin to be correctly set (either manually or using our netlify.toml).

They also added this section though, which shows how to exactly do it via Github Action. I approved it and merged, but I do wonder if this is too much for this part of the docs, are we going too far / doing too much here. Since you are now redesigning deployment docs I wanted to hear what you think. Maybe we will have a specific Github Actions section and this will be better explained there directly, instead of here under Netlify? Should we matbe (re)move this part?

Reply:
@Martinsos @infomiho Just wanted to hear your opinion on something similar: I am interested in writing documentation to deploy OpenSaaS on AWS (either EC2 or Lightsail), probably also including GitHub Actions to handle CI/CD.

I imagine I will briefly describe the peculiarities related to AWS, but maybe the GitHub Actions code is a bit too much? If that is the case I can put the detailed code that works with a particular version of OpenSaaS into a separate blog post, and only cover the general steps in the docs.

Reply:
@Bojun-Feng btw, @infomiho is on the conference from tomorrow so it might take him a bit longer to respond to this, but he should get to it the next week latest.

As for your question: Ok nice, that would be cool, the docs for AWS! Btw why OpenSaas, why not Wasp? I don't think there is anything OpenSaas specific? Or is there?

I think you are right, Github Action might be too much, especially if it has a lot of stuff that is "general" and not specific to AWS. But as you said, that could be a blog post for example, you are right! Btw we also ahve this https://github.com/wasp-lang/learning-materials -> sometimes we put stuff like this there. Hard to say at the moment if blog post or this is better, @infomiho might have some good ideas.